### PR TITLE
fixes to the new animal form

### PR DESCRIPTION
**Pull request by tnguyen3645**
Changes:

- Added space after private
- Removed controller tests
- Renamed show spec test to new animal page spec test
- Refactored Router file
- Refactored routes.rb

### CODE REVIEW

File: app/javascript/react/components/AnimalIndex.js [hunk 1]
```diff
@@ -25,8 +25,8 @@ const AnimalIndex = (props) => {
 
   const animalList = animals.map((animal) => {
     return (
-      <div className="cell small-6 medium-4">
-        <AnimalTile key={animal.id} id={animal.id} name={animal.name} />
+      <div key={animal.id} className="cell small-6 medium-4">
+        <AnimalTile id={animal.id} name={animal.name} />
```
Comment (kaydc18) [inline app/javascript/react/components/AnimalIndex.js:29]: You actually want this div to be inside of animal tile if you're going to add a key there. It's best practice for keys to be delivered inside a component.

**Review by kaydc18 — CHANGES_REQUESTED** [review]
Looks much better! One more thing that should be changed

**Comment by tnguyen3645** [comment]
Changes:

- moved key prop into AnimalTile
- moved div into Animal Tile component

**Review by kaydc18 — APPROVED** [review]
BOOYA!

**Pull request merged**